docs/reference/commandline/rm.md

```diff
@@ -69,7 +69,7 @@ The main process inside the container referenced under the link `redis` will rec
 ### Remove all stopped containers
 
 ```bash
-$ docker rm $(docker ps -a -q)
+$ docker rm (docker ps -a -f "status=exited" -q)
```

Comment:
The `$` should stay here, otherwise it won't work on Bash (when using Fish, the `$` should not be used though.